metabase/metabase
# Fix pie rows settings include filtered out values

Closes https://github.com/metabase/metabase/issues/47867

### Description

After editing pie rows settings, if dataset returned fewer rows we still showed previously existing rows in the settings sidebar. This PR filters them out.

### How to verify

- Create a Pie chart, reorder rows
- Apply filter or row limit to remove some rows from the dataset
- Ensure the settings sidebar shows rows that only exist in the dataset

### Checklist
